iidm/iidm-xml-converter/src/test/resources/slackTerminal.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need to add copyright header in this file too?

Reply:
Maybe not, there is no copyright in the network xml files for the other extensions.